pandas/core/arrays/categorical.py

```diff
@@ -906,7 +906,7 @@ def set_categories(
            Whether or not to reorder the categories in-place or return a copy
            of this categorical with reordered categories.
 
-           .. deprecated:: 1.3.0
+           .. Deprecated:: 1.3.0
```

Comment:
we can revert this now we've checked it :)

Reply:
yes I checked it on my side- not sure why it appeared here as I corrected it :woman_shrugging:

Reply:
also you can now remove this part of the check from `ci/code_checks.sh`

Reply:
> not sure why it appeared here as I corrected it

did you do `git add -a` and `git commit -m '<some message>'` after you'd corrected it?